File: client-app/src/app/models/emailactivity.ts
export interface EmailActivity {
    id: string;
    title: string;
    date: Date | null;
    description: string;
    city: string;
    venue: string;
  }


  export class EmailActivity implements EmailActivity {
    constructor(init?: EmailActivityFormValues){
      Object.assign(this,init)
    }
  }

  export class EmailActivityFormValues {
    id?: string = undefined;
    title: string= '';
    description: string= '';
    date: Date | null = null;
    city: string = '';
    venue: string = '';

    constructor(activity?: EmailActivityFormValues){
      if(activity) {
        this.id = activity.id;
        this.title = activity.title;
        this.description = activity.description;
        this.date = activity.date;
        this.venue = activity.venue;
        this.city = activity.city;
      }
    }
  }
  

File: Domain/emailActivity.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Threading.Tasks;

namespace Domain
{
    public class EmailActivity
    {
        public Guid Id { get; set; }//gjenerimi i id nga db

        public string Title { get; set; }

        public DateTime Date { get; set; }

        public string Description { get; set; }

        public string City { get; set; }

        public string Venue { get; set; }

        public AppUser userId { get; set; }

    }
}